_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -2753,6 +2809,10 @@ mfxStatus VAAPIEncoder::Execute(
                                                           m_vaContextEncode, m_maxFrameSizeId), MFX_ERR_DEVICE_FAILED);
     configBuffers[buffersCount++] = m_maxFrameSizeId;
 
+    MFX_CHECK_WITH_ASSERT(MFX_ERR_NONE == SetMultiPassFrameSize(task, fieldId, m_vaDisplay,
```

Comment:
Also, please, put this function call out of the macro. It's expansion in debug build will lead to double function call. I suggest:
```
sts = SetMultiPassFrameSize(task, fieldId, m_vaDisplay,m_vaContextEncode, m_multiPassFrameSizeId);
MFX_CHECK_WITH_ASSERT(sts == MFX_ERR_NONE, MFX_ERR_DEVICE_FAILED);
```

Reply:
Will move the function call out of the macro.